components/meta-packages/build-essential/build-essential.p5m

```diff
@@ -43,7 +43,7 @@ depend fmri=developer/sunstudio12u1 type=require
 
 # Headers
 depend fmri=system/header type=require
-depend fmri=system/library/math/header-math type=require
+depend fmri=system/library/math type=require
```

Comment:
You are right.  The system/library/math is not needed since it is already included by indirect dependencies.